<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notifications</class>
 <widget class="QDialog" name="notifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="notifications_bttn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_red_mark">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="logout_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="feed_btn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Feed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>130</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Notifications</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_notifications">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_handle">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Handle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profile_btn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
  </widget>
  <zorder>notifications_bttn</zorder>
  <zorder>logout_btn</zorder>
  <zorder>feed_btn</zorder>
  <zorder>search_btn</zorder>
  <zorder>listWidget_notifications</zorder>
  <zorder>pushButton_handle</zorder>
  <zorder>profile_btn</zorder>
  <zorder>label_red_mark</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
